src/Compiler.php

```diff
@@ -214,6 +214,7 @@ protected function renderRelations(Registry $registry, Entity $entity): array
     }
 
     /**
+     * @deprecated
```

Comment:
What can we use instead?

Reply:
We should use `$entity->getPrimaryFields()` instead.
We can add description for this deprecation, for example: `use $entity::getPrimaryFields() method.`